src/CMakeLists.txt

```diff
@@ -1425,6 +1425,7 @@ set( AUDACITY_LIBRARIES
    lib-time-track-interface
    lib-snapping-interface
    lib-audio-io-interface
+   lib-stretching-sequence-interface
```

Comment:
Still an unnecessary arc in the modules graph

Reply:
addressed